examples/todo-backend/todo-backend-bootable-jar.yaml

```diff
@@ -26,7 +26,7 @@ deploy:
       valueFrom:
         secretKeyRef:
           key: database-password
-          name: todos-db
+          name: todo-backend-db
```

Comment:
this is not correct. The `database-password' key comes from the `todos-db` template that is used to create the PostgreSQL database

Reply:
oups, wrong line